ra10ke.gemspec

```diff
@@ -22,4 +22,10 @@ Gem::Specification.new do |spec|
   spec.add_dependency "git"
   spec.add_dependency "solve"
   spec.add_dependency 'semverse', '~> 2.0'
+  spec.add_dependency 'table_print'
+  spec.add_development_dependency 'rubocop'
```

Comment:
we made the experience that rubocop heavily changes between minor and even patch releases. Should we pin the gem?

Reply:
Are we concerned with a particular ruby version?  I would say we should be consistent with other vox gems.

Reply:
I just removed the dev dependencies for now.